std/assembly/arraybuffer.ts

```diff
@@ -6,31 +6,28 @@ import { E_INVALIDLENGTH } from "./util/error";
 
 export abstract class ArrayBufferView {
 
-  @unsafe data: ArrayBuffer;
-  @unsafe dataStart: usize;
-  @unsafe dataLength: u32;
+  @unsafe readonly dataStart: usize;
```

Comment:
Wondering if there is a particular reason for changing the order here? Asking because if it remains this way, there's a constant in the loader that needs to be updated.

Reply:
the main reason is `dataStart` used more frequent then `buffer` so I guess have pure `load` without `offset=4` will be better. I'll fix loader

Reply:
Hmm, since user code will solely access `.buffer`, I guess this depends on the use case? Like, if user code doesn't access `.buffer`, there'll be more `.dataStart`, but if it does, there might be a lot more `.buffer` at some point. Do you know if a load with an offset has implications other than code size?

Reply:
No it doesn't affect to codesize. Just on codegen:
```asm
mov eax, dword ptr [r15 + rdi] 
```
vs
```asm
mov eax, dword ptr [r15 + rdi + 4]
```

Reply:
I see, though I still wonder what's the advantage then. I'm a bit concerned here because not every existing program might use the loader, and do manual loads expecting that `buffer` and `dataStart` are at specific offsets instead, so these programs would break with the change.

Reply:
Ok, I'll revert this changes

Reply:
Done!